lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -132,4 +146,43 @@ export abstract class BaseManagedIdentitySource {
             managedIdentityRequest
         );
     }
+
+    private async retryWithWWWAuthenticate(
```

Comment:
This is Azure ARC specific, it should not be in base class?

Reply:
Yes this is specific to azure arc.